News application - Pinned news (#160)
exoplatform/exo-documentation: docs/GettingStarted.rst

|image191|

> News application views

Reviewer: I would use `filters` instead of `views`. WDYT ?

Author: I wrote Views to distinguish between filtering by spaces and the different views availables from the dropdown menu as it is not the same used component.

Reviewer: Yes, but functionally speaking it is still filters. Talking about views make me think of different layouts for the News list.

Author: OK. I updated the text

^^^^^^^^^^^^^^^^^^^^^^^

> Different views are available from News application so you can easily find pinned articles, your posted articles, your scheduled articles, your drafts or your archived articles.

Reviewer: You anticipate, right ? :)

Author: Yes. To have the intro for the next documentation PRs.

> To display the pinned news list, all you have to do is to choose this option from the dropdown menu of the application topbar.

Reviewer: ```suggestion
To display the pinned news list, select this option from the dropdown menu of the application topbar.
```
